website/src/data/users.tsx

```diff
@@ -549,6 +549,14 @@ const Users: User[] = [
     source: 'https://github.com/DigiPie/kaya-folio',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: 'FAST',
+    description: 'The adaptive interface system for modern web experiences.',
+    preview: require('./showcase/fast.png'),
+    website: 'https://www.fast.design/',
+    source: 'https://github.com/microsoft/fast/tree/master/sites/website',
+    tags: ['opensource', 'product', 'design', 'large'],
```

Comment:
doc is not particularly large + the design is not really related to Docusaurus itself